docs/export/export.md

```diff
@@ -631,10 +638,135 @@ We list here a history of the calling convention version numbers:
     and the default since February 1st, 2024 (JAX 0.4.24).
     This is the only supported version as of 27th of March, 2024.
 
+## Developer documentation
+
+(export_debugging)=
+### Debugging
+
+You can log the exported modules, with somewhat different flags in OSS versus
+in Google. In OSS you can do the following:
+
+```shell
+# Log from python
+python tests/export_test.py JaxExportTest.test_basic -v=3
+# Or, log from pytest to /tmp/mylog.txt
+pytest tests/export_test.py -k test_basic --log-level=3 --log-file=/tmp/mylog.txt
```

Comment:
I think you can also do `tests/export_test.py::JaxExportTest::test_basic`.

Reply:
Yes, but `-k` is shorter, and it is not important for this example.